tech/languages/dotnet/dotnet.md

```diff
@@ -0,0 +1,40 @@
+---
+title: .NET
+subsection: dotnet
+section: tech-languages
+order: 2
+version: 7.8.4
+description: ".NET Core and Mono open source frameworks, and available IDEs."
+---
+
+## .NET Core Installation
+
+Enable the .NET SIG's copr repository:
+```
+$ sudo dnf copr enable @dotnet-sig/dotnet
+```
+
+Install the latest SDK:
+```
+$ sudo dnf install dotnet
+```
+
+Or a specific version:
+```
+$ sudo dnf install dotnet-sdk-2.1
+```
+
+To install runtime only, for example to merely deploy already prebuilt applications, where _x_ stands for major and _y_ stands for minor version:
+```
+$ sudo dnf install dotnet-runtime-x.y
+```
+
+Preview packages can be installed after enabling the preview copr repository:
```

Comment:
That would be contrary to what _we_ say though, we can't say "this is our recommended source for dotnet packages built by our team of engineers" and then when you get there, you read "this is not supported, use at your own risk."

Reply:
...technically it is supported by our engineers (the packages are, availability of the copr repository is a whole another story...)

Reply:
Well, the thing is that it's not the same as Fedora. It's more like a personal repository. It would be really nice if you could just link the copr page.